web/package.json

```diff
@@ -16,16 +16,20 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@next/font": "^13.1.5",
-    "ejs": "^3.1.8",
```

Comment:
I missed this change. Going to need to implement CI/CD to prevent mistakes like this (inadvertently dropping a dependency).